e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -15,13 +23,7 @@ export interface Context {
     isPlatformAdmin: boolean;
   };
   team: { id?: number } & NewTeam;
-  flow?: {
-    id?: string;
-    publishedId?: number;
-    slug: string;
-    name: string;
-    data?: object;
-  };
+  flows?: Flow[];
```

Comment:
This was the main change to `flows` so that the context takes in an array of flows instead of one flow object.